Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
@tianon The test failed because frame.Package is now _/go/src/github.com/opencontainers/runc/libcontainer/stacktrace instead of github.com/opencontainers/runc/libcontainer/stacktrace (in libcontainer/stacktrace/capture_test.go).

Re .gitignore, sure, though would still prefer not to tell git to ignore files which are tracked ;)